docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -0,0 +1,153 @@
+3. **Deployment**: Should include a deployment diagram which in turn explains components and how they interact with databases and each other.
+
+
+### Conceptual guides
+
+**Conceptual guides** are a continuations of our architectural guides and distinct from how-to guides. They explain a concept that is relevant for understanding Teleport. Some overlap will be likely between conceptual guides and architecture guides.
+
+### Reference manuals
+
+**Reference manuals** provide an exhaustive list of configuration options, API methods, and other possible fields and values for the various ways in which users can interact with Teleport.
+
+#### Expectations
+- Should be comprehensive. If listing configuration options, API paths, and so on, list all of them, rather than a few examples.
+- HTML or Markdown tables are often the best formats for this kind of article .
+- Should be easy to navigate with a browser's search functionality. List all content on the same page.
+- Should avoid prose and be expressed with brevity.
+
+
+## General style rules
+Please refer to this style guide when determining how address questions about English grammar, usage, and so on. Since many of these rules have equally justifiable alternatives, a style guide prevents debates over these questions from getting in the way of documenting Teleport. 
+
+### Body text
+- Write sparser one to two-sentence groupings rather than lengthier blocks of text.
+- Use periods at the end of a line even in a list unless the ending item is a command.
+
+### Code, commands, and configuration
+- `tsh`, `tctl`, and other core commands should be placed in backticks.
+- All ports or values should be enclosed in backticks, e.g., `443`.
+- Prefer putting commands (e.g., Bash scripts) into full-line code snippets. These will render with a handy copy button.
+
+   ```code
+   $ echo "This is an example."
+   ```
+
+### Diagrams
+- Use [Teleport's lucidchart library](https://app.lucidchart.com/lucidchart/dfcf1f4a-5cf0-4758-8ebb-f6ea86900aba/edit) to create diagrams with a consistent design language.
+- Diagram multistep sections using sequence diagrams that depict steps linearly.
+- Several great examples are available here: https://gravitational.slab.com/posts/diagrams-ix9nzhpd.
+
+### Footnotes
+- Footnotes should be ordered by appearance (chronological precedence). `2` should not come before `1`.
+
+### Headings
+- Headings should be in sentence case. For example, "Next steps" is preferred over "Next Steps." We want to clarify proper nouns and products in headings by using sentence casing.
+
+### Lists
+- We prefer short paragraph blocks over bulleted or numbered lists. This leads to a preference for completeness and brevity rather than enumeration. 
+- When you do include a list, we prefer bullet points over numberd lists.
+- Use numbered lists for any sequence of steps, but use these sparingly.
```

Comment:
IIRC, @klizhentas prefers `1/N` notation for a sequence of steps, so readers know how far they are in the process. Do we want to mention that here?